pkg/host/internal/sriov/sriov.go

```diff
@@ -311,6 +311,63 @@ func (s *sriov) DiscoverSriovDevices(storeManager store.ManagerInterface) ([]sri
 	return pfList, nil
 }
 
+func (s *sriov) DiscoverSriovVirtualDevices() ([]sriovnetworkv1.InterfaceExt, error) {
```

Comment:
Can we rename this to `DiscoverSriovVirtualPF`?

Reply:
I change it to `DiscoverSriovVirtualDevices` let me know if that is fine